Makefile

```diff
@@ -98,30 +118,9 @@ integration-test: docker-test
 	# sudo docker run --rm -it quay.io/fossa/fossa-cli-test
 
 # Release tasks.
-.PHONY: prepare-release
-prepare-release:
-	if [ -z "$$RELEASE" ]; then exit 1; fi
-	mv install.sh install.prev.sh
-	make -s installer > install.sh
-	git add install.sh
-	git commit -m "release($(RELEASE)): Release version $(RELEASE)" || make abort-release
-	git tag $(RELEASE) || (git reset HEAD^ && make abort-release)
-	rm install.prev.sh
+install.sh: $(GODOWNLOADER)
+	godownloader --repo=fossas/fossa-cli > install.sh
 
 .PHONY: release
-release:
-	if [ -z "$$(git tag -l --points-at HEAD)" ]; then exit 1; fi
-	[ "$$(grep "^  RELEASE='$$(git tag -l --points-at HEAD)'$$" install.sh | wc -l)" = "1" ]
+release: $(GORELEASER) install.sh
```

Comment:
Is goreleaser using install.sh here? If not, do we still need install.sh as a prerequisite?

Reply:
It's not, but this ensures that the file exists.